lib/fastlane/actions/scp.rb

```diff
@@ -0,0 +1,108 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class ScpAction < Action
+
+      def self.run(params)
+        Actions.verify_gem!('net-scp')
+        require "net/scp"
+        ret = nil
+        Net::SCP.start(params[:host], params[:username], {port: params[:port].to_i, password: params[:password]}) do |scp|
+          if params[:upload]
+            scp.upload! params[:upload][:src], params[:upload][:dst], recursive: true
+            Helper.log.info ['[SCP COMMAND]', "Successfully Uploaded", params[:upload][:src], params[:upload][:dst]].join(': ')
+          end
+          if params[:download]
+
+            t_ret = scp.download! params[:download][:src], params[:download][:dst], recursive: true
+            Helper.log.info ['[SCP COMMAND]', "Successfully Downloaded", params[:upload][:src], params[:upload][:dst]].join(': ')
+            unless params[:download][:dst]
+              ret = t_ret
+            end
+          end
+        end
+        ret
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Transfer files via SCP"
+      end
+
+      def self.details
+        "Allows SCP file transfers"
```

Comment:
This doesn't add any extra information. Please either provide more information about how to use it, or remove this string